crates/analyzer/src/code.rs

```diff
@@ -148,6 +148,7 @@ pub enum IssueCode {
     NoValidCatchTypeFound,
     NoValue,
     NonClassUsedAsAttribute,
+    NonDocumentedMethod,
     NonExistentAttributeClass,
```

Comment:
This file should not be edited manually.

Please rebase, add the code to [the script](https://github.com/carthage-software/mago/blob/fb090217a8c4f4b7c7360f24e6822a0d4983dfef/scripts/regen-analyzer-issue-codes.php#L56) and run `just regen-analyzer-issue-codes`.